firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -311,6 +311,9 @@ object Serializer {
       writers.foreach { w => b ++= "writer => "; b ++= legalize(w); newLineAndIndent(1) }
       readwriters.foreach { r => b ++= "readwriter => "; b ++= legalize(r); newLineAndIndent(1) }
       b ++= "read-under-write => "; b ++= readUnderWrite.toString
+    case DefTypeAlias(info, name, tpe) =>
+      b ++= "type "; b ++= name; b ++= " = ";
+      s(tpe) //; s(info) TODO: Uncomment once firtool accepts infos for type aliases
```

Comment:
While it's rather minor, this is not in the spec and AFAIK not planned? Or is that something you/someone is wanting to change (and make useful)? Or are there other paths that would be consuming this where it's not useful or relevant what spec says or something?

Reply:
Jack expressed that it would be useful for the FIRRTL spec to include the source info as part of a type alias, as it would allow us to locate what Chisel source line generated a particular alias, since there isn't exactly a one-to-one correspondence with a FIRRTL type alias and its associated Chisel `Record`

Reply:
My main thinking is that in emitting structs in the SystemVerilog it would be really good to have a source locator describing where the type was defined. It's not crucial but would be good.

Reply:
I agree that more source infos the better...